wizeline/wize-q-remix
# refactor(utils): remove unused utils files and restructure utils directory

#### What does this PR do? 
- Remove unusued utils files and organizes utils directory, removing the "backend" folder remaining from the migration and moving it's content to other appropiate directory.

#### How should this be manually tested?
1. Build the project and assert it compiles `npm run build`
2. Run the project as usual and go through some features. npm run build should pick up reference errors though maybe thats enough.

#### Any background context you want to provide?
1. Only file location references were changed, functionality was left intact.
2. It is a big PR diff, most being file reference changes

#### What are the relevant tickets?
Closes #232
